bin/reth/src/optimism.rs

```diff
@@ -14,7 +14,7 @@ use std::sync::Arc;
 #[global_allocator]
 static ALLOC: tikv_jemallocator::Jemalloc = tikv_jemallocator::Jemalloc;
 
-#[cfg(not(feature = "optimism"))]
+#[cfg(all(not(feature = "optimism"), not(feature = "telos")))]
```

Comment:
I see you also removed a similar change from `bin/reth/src/main.rs`, do we need either of these?  If we always build with the `telos` feature flag, should we need to worry at all about the other binaries besides our own `bin/reth/src/telos.rs`?  If possible, neither of these changes would be ideal, so we have fewer changes/merge conflicts.

Reply:
not sure I'm following, you think that its OK to make it compilable only with our feture flag, and doing that way will reduce differences?